kano_updater/status.py

```diff
@@ -77,6 +78,7 @@ def load(self):
                 data['last_update']
                 data['last_check']
                 data['last_check_urgent']
+                data['first_boot_countdown']
```

Comment:
Yup, the field was added just like when we added `last_check_urgent`, `is_shutdown` etc.
It worked fine for me. Did you test it?